lisp/ess-comp.el

```diff
@@ -37,6 +37,8 @@
 (add-to-list 'load-path nil)
 
 ;; defvar'ed to nil in ./ess-site.el
+;; TODO: ^^^^ This isn't true?
```

Comment:
ESS used to depend on ess-site but I've been trying to remove that dependency. The defvar here looks like the right thing to do.

@mmaechler Could we make the value of `ess-show-load-messages` during compilation dependent on an environment variable? I think those load messages are distracting.

Reply:
Agreed. We should remove dependency on ess-site and streamline inner dependencies. The everything depends on everything is not the right approach.